salt/modules/elasticsearch.py

```diff
@@ -174,7 +174,7 @@ def alias_get(indices=None, aliases=None, hosts=None, profile=None):
     return None
 
 
-def document_create(index, doc_type, body=None, hosts=None, profile=None):
+def document_create(index, doc_type, body=None, id=None, hosts=None, profile=None):
```

Comment:
Okay, I reviewed the other methods and it looks like `id` is everywhere it should be